packages/dev-server-storybook/src/serve/storybookPlugin.ts

```diff
@@ -30,7 +30,7 @@ export function storybookPlugin(pluginConfig: StorybookPluginConfig): Plugin {
   return {
     name: 'storybook',
 
-    serverStart(args: { config: unknown }) {
+    serverStart(args: { config: DevServerCoreConfig }) {
```

Comment:
I got this error after adding "types" for "dev-server-core", so needed to fix it here:

```
> build
> npm run build:clean && tsc --build


> build:clean
> rimraf --glob packages/*/tsconfig.tsbuildinfo && rimraf --glob packages/*/dist

packages/dev-server-storybook/src/serve/storybookPlugin.ts:34:7 - error TS2322: Type 'unknown' is not assignable to type 'DevServerCoreConfig'.

34       serverConfig = args.config;
         ~~~~~~~~~~~~


Found 1 error.
```